src/asm/fstack.cpp

```diff
@@ -309,6 +310,20 @@ void fstk_RunInclude(std::string const &path, bool preInclude) {
 		fatalerror("Failed to set up lexer for file include\n");
 }
 
+std::set<std::string> includedOnce;
```

Comment:
Move this to the top with the other variable declarations.

Reply:
(Also, please mark it as `static` to improve codegen, since it's only used from within `fstack.cpp`.)

Reply:
This variable declaration has been moved to the top of the file, and made static.